cli/app.go

```diff
@@ -29,9 +29,15 @@ const (
 	// TODO: RSDK-6683.
 	quietFlag = "quiet"
 
-	logsFlagErrors = "errors"
-	logsFlagTail   = "tail"
-	logsFlagCount  = "count"
+	logsFlagFormat     = "format"
+	logsFlagOutputFile = "output"
+	logsFlagKeyword    = "keyword"
+	logsFlagLevels     = "levels"
+	logsFlagStartTime  = "start"
+	logsFlagEndTime    = "end"
```

Comment:
(minor) I'm gonna ask you to do a bit of bookkeeping for us here if you don't mind! We already have a "start" flag and an "end" flag (`dataFlagStart` and `dataFlagEnd`). Would you be willing to rename those to `generalFlagStart` and `generalFlagEnd` and then update call sites so that there's only the single instance of a `--start` or `--end` flag?